src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp.mdx

```diff
@@ -0,0 +1,283 @@
+
+## Configure the endpoint, port and protocol [#endpoint-port-protocol]
+
+Requirement level: **Required**
+
+In order to configure send OTLP data to New Relic, you must configure your OTLP exporter to use the relevant endpoint and port from the table below based on your environment.
+
+The mechanism to configure the endpoint will vary, but OpenTelemetry language SDKs generally support setting the `OTEL_EXPORTER_OTLP_ENDPOINT=<INSERT_ENDPOINT>` environment variable (see [OpenTelemetry docs](https://opentelemetry.io/docs/specs/otel/protocol/exporter/) for more info).
+
+Additionally, you should configure your OTLP exporter to use the [OTLP/HTTP binary protobuf version](https://opentelemetry.io/docs/specs/otlp/#binary-protobuf-encoding) of the protocol if available. While New Relic supports all versions of OTLP, OTLP/HTTP binary protobuf has proved to be more robust than gRPC without any apparent reduction in performance.
+
+The mechanism to configure the endpoint will vary, but OpenTelemetry language SDKs generally support setting the `OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf` environment variable (see [OpenTelemetry docs](https://opentelemetry.io/docs/specs/otel/protocol/exporter/) for more info).
+
+If using the collector, prefer the [otlphttpexporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter).
+
+<table>
+  <thead>
+    <tr>
+      <th>
+        Environment
+      </th>
+
+      <th>
+        gRPC
+      </th>
+
+      <th>
+        HTTP
+      </th>
+
+      <th>
+        Endpoint
+      </th>
+
+      <th>
+        Supported Ports
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        US OTLP
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        `https://otlp.nr-data.net`
+      </td>
+
+      <td>
+        `443`, `4317`, `4318`
+      </td>
+
+    </tr>
+
+    <tr>
+      <td>
+        EU OTLP
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        `https://otlp.eu01.nr-data.net`
+      </td>
+
+      <td>
+        `443`, `4317`, `4318`
+      </td>
+
+    </tr>
+
+    <tr>
+      <td>
+        US FedRAMP OTLP<br/>(See [FedRAMP compliance](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints/#otlp-api) for more information)
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        ✅
+      </td>
+
+      <td>
+        `https://gov-otlp.nr-data.net`
+      </td>
+
+      <td>
+        `443`, `4317`, `4318`
+      </td>
+
+    </tr>
+
+    <tr>
+      <td>
+        Infinite Tracing<br/>(See [best practices](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts/#infinite-tracing)  for endpoint details
```

Comment:
@jack-berg This link redirects to the wrong page (and a page I need to remove). I couldn't find other content about infinite tracing in the OTEL docs. Is there a page or section you had in mind for this link?